client/client.go

```diff
@@ -76,21 +76,21 @@ type Client interface {
 	BuildQueue() ([]*model.Feed, error)
 
 	// BuildStart re-starts a stopped build.
-	BuildStart(string, string, int) (*model.Build, error)
+	BuildStart(string, string, int, []string) (*model.Build, error)
```

Comment:
edit: maybe instead this should be `map[string]string` and the caller should be in charge of converting to the `map[string]string` format.

the reason is that this client package can be imported and used by programs that want to interact with drone, and using `map[string]string` seems more declarative that using `[]string` and having to document `key=value` format for each item in the slice